specification/blueprint/resource-manager/Microsoft.Blueprint/preview/2018-11-01-preview/blueprintAssignment.json

```diff
@@ -139,6 +139,24 @@
             "description": "No Content"
           }
         }
+      },
+      "post": {
+        "operationId": "WhoIsBlueprint",
+        "description": "Get Blueprints service SPN objectId",
+        "x-ms-examples": {
+          "WhoIsBlueprint_Action": {
+            "$ref": "./examples/WhoIsBlueprint_Action.json"
+          }
+        },
+        "parameters": [],
```

Comment:
This really doesn't take any parameters?

Reply:
Yeah. It gets the tenant ID from headers provided by ARM.

---
In reply to: [268895118](https://github.com/Azure/azure-rest-api-specs/pull/5470#discussion_r268895118) [](ancestors = 268895118)

Reply:
Actually this is going to be a problem.  There are two path parameters in the URL that must be provided.  See the validation failure [here](https://travis-ci.org/Azure/azure-rest-api-specs/jobs/511258079).